test/NuGet.Clients.Tests/NuGet.PackageManagement.VisualStudio.Test/ProjectSystems/TestVSProjectAdapter.cs

```diff
@@ -0,0 +1,217 @@
+using System;
+using System.Collections.Generic;
+using System.IO;
+using System.Linq;
+using System.Runtime.Versioning;
+using System.Text;
+using System.Threading.Tasks;
+using EnvDTE;
+using Microsoft.VisualStudio.Shell.Interop;
+using Moq;
+using NuGet.Frameworks;
+using NuGet.ProjectManagement;
+using NuGet.RuntimeModel;
+using NuGet.VisualStudio;
+
+namespace NuGet.PackageManagement.VisualStudio.Test
+{
+    public class TestVSProjectAdapter : IVsProjectAdapter
```

Comment:
nit: consider using `=>` in every one-line method in this class, to create inline function declarations.

Also, should this class be internal?

Reply:
yes to internal class and no to inline function